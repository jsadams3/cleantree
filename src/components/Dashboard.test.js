import React from "react";
import { shallow } from "enzyme";

import { Dashboard } from "./";

describe("Dashboard", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Dashboard />);
  });

  afterEach(() => {
    jest.clearAllMocks();
  });
  it("should render the component", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("should render the Header", () => {
    expect(wrapper.find("HeaderContainer").exists());
  });

  it("should render the Repositories", () => {
    expect(wrapper.find("RepositoriesContainer").exists());
  });
});
