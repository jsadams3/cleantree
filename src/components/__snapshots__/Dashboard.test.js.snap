// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard should render the component 1`] = `
<div>
  <HeaderContainer />
  <RepositoriesContainer />
</div>
`;
